pgcrypto/mixins.py

```diff
@@ -103,19 +99,44 @@ def _check_max_length_attribute(self, **kwargs):
         """Override `_check_max_length_attribute` to remove check on max_length."""
         return []
 
+    def get_col(self, alias, output_field=None):
+        """Get the decryption for col."""
+        if output_field is None:
+            output_field = self
+        if alias != self.model._meta.db_table or output_field != self:
+            return DecryptedCol(
```

Comment:
@peterfarrell I just noticed that this one statement is the only statement not covered by the tests. It seems a fairly significant statement to cover, so would be worth adding a test if you have time.

Reply:
Yea, honestly I've been trying to figure out how to trigger that section of code. It's there because we need to cover that (according to the code in Django). Any thoughts on how to trigger that?